### PR TITLE
Document updates

### PR DESCRIPTION
**Pull request by neilz-tt**
### Motivation

< The motivation for the changes in this PR. "Currently we...", "This is needed because..." >

### In this PR
* < Additions, removals, fixes, features >

< Ticket status, e.g. "fixes #issue number" > 

### Test Plan

< How to run the unit test, e2e test etc. >
< Or what are the actual request/response might look like (like grpcurl) >

### Future Work
* < Out of scope non-goals for this PR >
* < These should be links to tickets. If the tickets do not exist, make them. >

### CODE REVIEW

File: docs/v2/api-endpoints/sample_mixins.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ description: Sample a desired number of mixins from the ledger, excluding a list
 
 | Required Param | Purpose | Requirements |
 | :--- | :--- | :--- |
-| `outputs` | The TXOs to get the membership proofs for | TXO must exist in the ledger |
+| `num_mixins` | ??? | ??? |
+| `excluded_outputs` | ??? | ??? |
```
Comment (neilz-tt) [inline docs/v2/api-endpoints/sample_mixins.md:12]: what should be the proper Purpose and Requirements for these two parameters?

Comment (briancorbin) [reply docs/v2/api-endpoints/sample_mixins.md:12]: | `num_mixins` | The number of mixins to sample | Must be less than the number of txos in the ledger minus number of excluded outputs |
| `excluded_outputs` | Txos to exclude from sampling | Txo must exist in the ledger |

Comment (neilz-tt) [reply docs/v2/api-endpoints/sample_mixins.md]: got it, thank you!

**Review by briancorbin — APPROVED** [review]
Thanks for this! :)

**Pull request merged**